modules/install-ibm-cloud-configuring-the-public-subnet.adoc

```diff
@@ -0,0 +1,160 @@
+// This is included in the following assemblies:
+//
+// installing_ibm_cloud/install-ibm-cloud-installing-on-ibm-cloud.adoc
+
+[id="configuring-the-public-subnet_{context}"]
+= Configuring the public subnet
+
+All of the {product-title} cluster nodes must be on the public subnet. IBM Cloud&#174; does not provide a DHCP server on the subnet. Set it up separately on the provisioner node. This document describes configuring Dnsmasq on the public subnet.
+
+.Procedure
+
+. Ensure `dnsmasq` is running:
++
+[source,terminal]
+----
+$ systemctl status dnsmasq
+----
++
+If `dnsmasq` isn't running, start it and enable it:
++
+[source,terminal]
+----
+$ sudo systemctl start dnsmasq
+----
++
+[source,terminal]
+----
+$ sudo systemctl enable dnsmasq
+----
+
+. Set an unused IP address from the public IP range:
++
+[source,terminal]
+----
+$ UNUSEDIP=<IP-address>
+----
+
+. Make a directory for the Dnsmasq configuration files:
++
+[source,terminal]
+----
+$ sudo mkdir -p  /var/lib/baremetal-dhcp
+----
+
+. Open the Dnsmasq configuration file:
++
+[source,terminal]
+----
+$ sudo vi /var/lib/baremetal-dhcp/baremetal-dhcp.conf
+----
+
+. Add the following configuration to the Dnsmasq configuration file:
++
+[source,configuration files]
+----
+interface=baremetal
+except-interface=lo
+bind-dynamic
+log-dhcp
+
+dhcp-range=$UNUSEDIP,$UNUSEDIP,$PUBLICCIDR
+dhcp-option=baremetal,121,0.0.0.0/0,$BAREMETAL_GATEWAY,$BOOTSTRAP_PRIV_IP,$BSHOSTIP
+
+pid-file=/var/lib/baremetal-dhcp/baremetal-dhcp.pid
+dhcp-hostsfile=/var/lib/baremetal-dhcp/baremetal-dhcp.hostsfile
+addn-hosts=/var/lib/baremetal-dhcp/baremetal-dhcp.addnhosts
+log-facility=/var/log/baremetal-dhcp.log
```

Comment:
These 4 lines can be replaced with
dhcp-hostsfile=/var/lib/dnsmasq/dnsmasq.hostsfile

Reply:
done